app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
Edit and create are the same forms? Why don't you use `partials`?

Reply:
Ok no problem

Reply:
@Jucerweb, My PR it's ready.